cephfs/admin/clone_test.go

```diff
@@ -27,6 +27,11 @@ var sampleCloneStatusInProg = []byte(`{
       "volume": "cephfs",
       "subvolume": "subvol1",
       "snapshot": "snap1"
+    },
+    "progress_report": {
+      "percentage cloned": "25%",
+      "amount cloned": "55/202",
+      "files cloned": "1/3"
```

Comment:
With the new test case added, I wouldn't bother adding these new fields to the existing sample which runs on every configured release version CI job.

Reply:
@phlogistonjohn Following my suggestion above, the latest version doesn't include the addition of new fields to the existing test. Do you want to add it back?